cadquery/occ_impl/shapes.py

```diff
@@ -428,17 +430,16 @@ def exportStl(
             Default is 1e-3, which is a good starting point for a range of cases.
         :param angularTolerance: Angular deflection setting which limits the angle between subsequent segments in a polyline. Default is 0.1.
         :param ascii: Export the file as ASCII (True) or binary (False) STL format.  Default is binary.
+        :param isToleranceRelative: If True, deflection used for discretization of each edge will be <tolerance> * <size of="" edge>="">. Deflection used for the faces will be the maximum deflection of their edges.
```

Comment:
Should this part of the docstring be `<tolerance> * <size of edge>` ? 

While there is no hard guideline on comment or docstring line length,  I would also suggest to add a few line breaks.  This is currently the longest docstring line in the file.

Reply:
Yeah I can shorten that down. I lazily copied it from the OCCT docs